app/controllers/api/v1/users_controller.rb

```diff
@@ -21,6 +21,8 @@ def create
       href: user_params["external_urls"]["spotify"],
       email: user_params["email"]
     )
+
+    session[:user_id] = @user.id
```

Comment:
added in this so I could use the `current_user` method within `application_controller`. Will probably cause merge conflicts.